RELEASE-NOTES.md

```diff
@@ -1,5 +1,50 @@
 # Prefect Release Notes
 
+## Release 2.7.11
+
+### Exciting New Features 🎉
+- Add refactor of results into a standalone artifact table — https://github.com/PrefectHQ/prefect/pull/8164
+- Add Prefect Cloud Events schema and clients. — https://github.com/PrefectHQ/prefect/pull/8357
+
+### Enhancements
+- Update pinned pip version in Docker images — https://github.com/PrefectHQ/prefect/pull/8289
+- Update default task run name to exclude hash of task key — https://github.com/PrefectHQ/prefect/pull/8292
+- Add functionality that updates preinstalled packages in Docker images — https://github.com/PrefectHQ/prefect/pull/8288
```

Comment:
"Add functionality" is weird here, perhaps?

- Update Docker images to update preinstalled packages on build